packages/traceloop-sdk/tests/datasets/test_datasets_operations.py

```
@@ -0,0 +1,119 @@
+
+    http = HTTPClient(
+        base_url="https://api-staging.traceloop.com",
+        api_key="invalid-key",
+        version="1.0.0",
+    )
+    invalid_datasets = Datasets(http)
+
+    try:
+        invalid_datasets.get_all()
+        # If this doesn't raise an exception, the test setup might be wrong
+        assert False, "Expected authentication error"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)